currency/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Simple Currency System
+Help Site: [This is Optional]/<> <-This is Not Optional
+
+There Will be More Commands In the Future :)
+
+Simple Currency System with Changeable Name/Emoji Money.
+
+It Works With Some Of The Fun Commands that work with Currency, ex: slotsMachine.cc.lua or guessNumber.cc.lua
+
+These Commands Are Standalone, You can add Only one, all or two
+
+# Some Current Commands It Have:
+1. **-balance [User/Member]**
+   It Gets The Current Balance of A User Member or if no arguments given, The Users Balance, ex: !balance
+1. **-addbalance <User/Member> <Amount>** 
+   It adds Balance To A User/Member, ex: !addbalance @Gas_Infinity 100
+1. **-rob <User/Member>** 

```

Comment:
```suggestion
1. **-balance [User/Member]**
   It Gets The Current Balance of A User Member or if no arguments given, The Users Balance, ex: !balance
2. **-addbalance <User/Member> <Amount>** 
   It adds Balance To A User/Member, ex: !addbalance @Gas_Infinity 100
3. **-rob <User/Member>** 
```
While, technically speaking, Markdown supports using `1. ... 1. .... 1. ...` for ordered lists, you should use an incrementing number instead - `1. ... 2. ... 3. ...`

Reply:
Changed